src/injected/analyzers/rule-analyzer.ts

```diff
@@ -11,11 +10,11 @@ import { ForRuleAnalyzerScanCallback } from '../../common/types/analyzer-telemet
 import { IScopingStoreData } from '../../common/types/store-data/scoping-store-data';
 import { ScanOptions } from '../../scanner/exposed-apis';
 import { ScanResults } from '../../scanner/iruleresults';
-import { IHtmlElementAxeResults, ScannerUtils } from '../scanner-utils';
+import { ScannerUtils } from '../scanner-utils';
+import { Analyzer, AxeAnalyzerResult, RuleAnalyzerConfiguration } from './analyzer';
 import { BaseAnalyzer } from './base-analyzer';
-import { AxeAnalyzerResult, IAnalyzer, RuleAnalyzerConfiguration } from './ianalyzer';
 
-export class RuleAnalyzer extends BaseAnalyzer implements IAnalyzer<IHtmlElementAxeResults> {
+export class RuleAnalyzer extends BaseAnalyzer implements Analyzer {
```

Comment:
`BaseAnalyzer` already implements `Analyzer`; not sure it's necessary to repeat `implements Analyzer` in each subclass now that you've removed the generic parameter

Reply:
(same applies to the other 5 similar instances)

Reply:
Good catch. Updated.